bin/internal/release-candidate-branch.version

```diff
@@ -1 +1 @@
-flutter-2.13-candidate.0
+3.0.5
```

Comment:
Please change this back to `flutter-2.13-candidate.0` once the tool is updated on your end, you will not have to do this again.

Reply:
modified it.